docs/src/main/paradox/play-framework.md

```diff
@@ -50,3 +54,58 @@ Scala
 Java
 :   @@snip[MyController.java]($root$/../play-interop-test-java/src/main/java/controllers/MyController.java) { #using-client }
 
+## Serving gRPC from a Play Framework app
+
+To be able to serve gRPC from a Play Framework app you must enable [HTTP/2 Support](https://www.playframework.com/documentation/2.6.x/AkkaHttpServer#HTTP%2F2-support-%28experimental%29)
+with HTTPS and the ALPN agent. (This is still somewhat involved and we hope to simplify it)
+
+Generating classes from the gRPC service definition is done buy adding the Akka gRPC plugin to your sbt build:
+
+sbt
+:   @@@vars
+    ```scala
+    // in project/plugins.sbt:
+    addSbtPlugin("com.lightbend.akka.grpc" % "sbt-akka-grpc" % "$projectversion$")
+    ```
+    @@@
+
+
+Then you need to enable the Play server side code generator in `build.sbt`:
+
+Scala
+:   ```scala
+enablePlugins(AkkaGrpcPlugin)
+import akka.grpc.gen.scaladsl.play.PlayScalaServerCodeGenerator
+akkaGrpcExtraGenerators += PlayScalaClientServerGenerator
+```
+
+Java
+:   ```scala
+enablePlugins(AkkaGrpcPlugin)
+import akka.grpc.gen.javadsl.play.PlayJavaServerCodeGenerator
+akkaGrpcExtraGenerators += PlayJavaClientServerGenerator
+```
+
+The plugin will look for `.proto` service descriptors in `src/main/protobuf` and output an abstract class per service
```

Comment:
Might be in `app/protobuf` instead of `src/main/protobuf`, e.g. https://github.com/raboof/play-scala-grpc-server/tree/master/app/protobuf, https://www.playframework.com/documentation/2.7.0-M2/Anatomy#Default-SBT-layout.

Reply:
I think Play docs (or migration guide) should encapsulate this particular info (not akka-grpc's).

Reply:
One idea we have discussed is to have the docs in the play docs. But since we have postponed doing that (and deciding to do that perhaps). Tracked in #300